source/core_http_client.c

```diff
@@ -1410,6 +1411,11 @@ static HTTPStatus_t addRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
     char rangeValueBuffer[ HTTP_MAX_RANGE_REQUEST_VALUE_LEN ];
     size_t rangeValueLength = 0U;
 
+    /* Directly passing in these strings to memcpy is a MISRA violation
+     * Due to this we allocate these pointers for MISRA compliance */
```

Comment:
same as above

Reply:
I thought we decided to mask the coverity warning here.

Reply:
There was already a precedent for adding in additional variables to remove MISRA violations, so that's what we decided to do here as well.